Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -126,15 +126,20 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
           .materialize()
       }
 
+    let allNewlyAddedCards: Signal<[UserCreditCards.CreditCard], Never> = newlyAddedCardProducer.values()
+      .scan(into: []) { array, card in
+        array.insert(card, at: 0)
```

Comment:
I forgot the `scan` at first - this turns a signal like `[a, b, c]` into a signal like `[[a], [a, b], [a, b, c]]`.